vermouth/pdb/pdb.py

```diff
@@ -443,6 +459,58 @@ def _do_single_conect(self, conect_record, id2idxs):
                             mol2.nodes[atomidx]['position'])
             mol.add_edge(atomidx0, atomidx, distance=dist)
 
+    def do_seqres(self):
+        """
+        Get the AA sequences of the chains in the molecules
+        """
+        chains = list(list(np.unique(list(mol.nodes[idx]['chain'] for idx in mol)))
+                      for mol in self.molecules)
+
+        properties = {a: [] for a in [x for xs in chains for x in xs]}
+
+        for line in self._seqres:
+            chain = line[11]
+            resnames = line[19:].split()
+            #some varient of this can check we have protein seqres and not
+            #seqres for nucleotides
+            #not much use atm but could be helpful for when we have nucleotides here
+            # try:
+            #     assert list(set(len(i) for i in resnames)) == 3
+            # except AssertionError:
+            #     #we don't really need to worry about this
+            #     #we just need to not use it any further
+            #     pass
+            properties[chain].append(resnames)
+
+        # this to make a flat list for each chain in the file
+        for chain in properties.keys():
+            properties[chain] = [x for xs in properties[chain] for x in xs]
+        self._check_seqres(properties)
+
+
+    def _check_seqres(self, properties):
+        '''
+        In future this could just be done by parsing REMARK 465.
```

Comment:
??

Reply:
see page 130 of the [pdb format file](https://files.wwpdb.org/pub/pdb/doc/format_descriptions/Format_v33_Letter.pdf) - missing residues are explicitly listed in the REMARK section of pdb files, but getting that will require parsing the whole REMARK section, which I think is well beyond anyone's interest atm...!

Reply:
Ah, right. And yes. Definitely out of scope for now!